enterprise_access/apps/content_assignments/models.py

```diff
@@ -361,6 +367,8 @@ def annotate_dynamic_fields_onto_queryset(cls, queryset):
                 LearnerContentAssignmentAction.objects.filter(
                     assignment=OuterRef('uuid'),
                     action_type=AssignmentActions.NOTIFIED,
+                    completed_at__isnull=False,
+                    error_reason__isnull=True,
```

Comment:
Before, this logic would consider notified actions even when they have an `error_reason` and/or have not been marked as "completed" via the `completed_at` field.